locobot/agent/tasks.py

```diff
@@ -77,15 +77,20 @@ def __init__(self, agent, task_data):
         super(Point, self).__init__()
         self.target = np.array(task_data["target"])
 
+    def get_pt_from_region(self, region):
+        assert len(region) == 6, "Region list has less than 6 elements (minx, miny, minz, maxx, maxy, maxz)"
+        return region[:3] # just return the min xyz for now
```

Comment:
take the mean, no?  probably doesn't matter that much

Reply:
this is an arbitrary choice. Going to keep it as the first xyz since the coords for objects output by locobot perception is approximately the centre of the object.